app/Main.hs

```diff
@@ -6,11 +6,11 @@ module Main
 where
 
 import           Control.Concurrent                       ( MVar
-                                                          , forkIO
                                                           , newMVar
                                                           , threadDelay
                                                           , tryPutMVar
                                                           )
+import qualified Control.Concurrent.Async                 as Async
```

Comment:
Why not skip the qualified import and simply import just `concurrently_` from Control.Concurrent.Async?

Reply:
I've taken to qualified imports since it's obvious at use site where the stuff comes from.

Reply:
I see your point, although I don't think you're missing that much when you list the imported stuff at the imports. 

My point was more that we should probably decide what we want to use so it's more or less consistent across the project :)

Reply:
But that is a discussion for somewhere else